packages/table/src/useLeafyGreenTable/useLeafyGreenTable.tsx

```diff
@@ -34,6 +36,16 @@ function useLeafyGreenTable<T extends LGRowData, V extends unknown = unknown>({
     (rest.initialState?.expanded as ExpandedState) ?? {},
   );
 
+  const prevColumns = usePrevious(columnsProp);
+  /**
+   * This is used to determine if table rows should re-render. If the column definitions are the same, table rows will not re-render. If they are different, all rows will re-render.
```

Comment:
```suggestion
   * This is used to determine if table rows should re-render when column definitions change. If the column definitions are the same, table rows will not re-render. If they are different, all rows will re-render.
```

Can you be explicit about what happens when this is true, and when it's false?